webviz_ert/controllers/plot_view_controller.py

```diff
@@ -13,6 +13,14 @@
 from webviz_ert.plugins._webviz_ert import WebvizErtPluginABC
 
 
+def _suffix_to_plot_type(suffix: str) -> str:
+    if suffix == "resp":
+        return "response"
+    if suffix == "param":
+        return "parameter"
+    raise Exception(f"Unexpected suffix `{suffix}`")
+
+
 def _new_child(parent: WebvizErtPluginABC, plot: Dict) -> Component:
```

Comment:
I think the `suffix` or hopefully the `DataType` is available when you call this function. If you added  it to the list of arguments the function definition would look something like

```
def _new_child(parent: WebvizErtPluginABC, plot_name: str, data_type: DataType) -> Component:
```

Reply:
and done completely with 59b97b9